protocol/messenger_handler.go

```diff
@@ -939,7 +973,7 @@ func (m *Messenger) HandleAcceptContactRequest(state *ReceivedMessageState, mess
 	return nil
 }
 
-func (m *Messenger) handleRetractContactRequest(contact *Contact, message protobuf.RetractContactRequest) error {
+func (m *Messenger) handleRetractContactRequest(messageState *ReceivedMessageState, contact *Contact, message protobuf.RetractContactRequest) error {
```

Comment:
I would avoid passing the whole `messageState`, makes things a bit harder to test, and I think eventually we want to get rid of it completely, looks like all you need is `Response`, I would just pass that, I think that's a bit nicer, but up to you

Reply:
fixed)